packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -10,6 +10,8 @@ import type {
 	RawDevConfig,
 	RawEnvironment,
 } from "../config";
+import { pipeline } from "node:stream";
```

Comment:
These imports are not used. I think you will need to run ESLint on this branch.

Reply:
Resolved all check:lint errors.